controller/users_blackbox_test.go

```diff
@@ -1509,6 +1511,25 @@ func (s *UsersControllerTestSuite) TestCreateUserAsServiceAccountWithRequiredFie
 	s.checkCreateUserAsServiceAccountOK(fmt.Sprintf("testuser%s@email.com", uuid.NewV4().String()))
 }
 
+func (s *UsersControllerTestSuite) TestCreateUserWithPreviewEmailsOK() {
```

Comment:
could you add a test to verify that the user creation request with same email address is rejected with the default configuration?